pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools>=61.2", "wheel"]
 
 [project]
 name = "polyapi-python"
-version = "0.3.3"
+version = "0.3.7.dev7"
```

Comment:
@FedeMarchiniHotovo right before you merge please remember to change this to `0.3.7`